### PR TITLE
ROCANA-5660: Rewrite netflow project imports

### PR DESCRIPTION
**Pull request by adamlamar**

### CODE REVIEW

File: cmd/nf-dump-pcap/main.go [hunk 1]
```diff
@@ -15,14 +15,14 @@ import (
 
 	"github.com/google/gopacket"
```
Comment (actgardner) [inline cmd/nf-dump-pcap/main.go]: Don't we also want to pin all the transitive dependencies? I think traditionally we've cloned everything.

Comment (adamlamar) [inline cmd/nf-dump-pcap/main.go]: I don't use anything under the `cmd` directory (they're just wrappers for cli programs), and `gopacket` isn't used elsewhere. The main repository doesn't complain when building, and I didn't see any other transitive dependencies.

Comment (actgardner) [inline cmd/nf-dump-pcap/main.go]: I'm pretty sure it's going to get imported by Jenkins when we run `go test ...` - if we don't use the `cmd` directory, can you remove it?

Comment (adamlamar) [inline cmd/nf-dump-pcap/main.go]: THERE ARE NO TESTS, BWA HAHAHA. But you're right, I removed `cmd`.

**Comment by adamlamar** [comment]
@alanctgardner Ready for another pass.

**Comment by actgardner** [comment]
LGTM :)

**Pull request merged**